index.js

```diff
@@ -27,10 +28,7 @@ function renderMathsExpression(expr) {
       displayStyle = true;
       expr = '\\displaystyle ' + expr.substr(1, expr.length - 2);
     }
-    var html = katex.renderToString(expr);
-    if (displayStyle) {
-      html = html.replace(/class=\"katex\"/g, 'class="katex katex-block" style="display: block;"');
-    }
+    var html = katex.renderToString(entities.decodeHTML(expr), displayStyle);

```

Comment:
I just realized this was incorrect, the displayStyle should be given in an options object. The line would be:
    var html = katex.renderToString(entities.decodeHTML(expr), { displayMode: displayStyle });